test/Fixtures.cmake

```diff
@@ -61,6 +61,12 @@ set(devilutionx_fixtures
   Levels/L4Data/Vile1.DUN
   Levels/L4Data/Warlord.DUN
   Levels/L4Data/Warlord2.DUN
+  timedemo/RogueLevel1to2/demo_0.dmo
+  timedemo/RogueLevel1to2/demo_0_reference_single_0.sv
+  timedemo/RogueLevel1to2/single_0.sv
+  timedemo/WarriorLevel1to2/demo_0.dmo

```

Comment:
I still would prefer if the playback was stored inside of the save (renamed to .dmo) to create a self contained file, that would also just show as bin in git.

Reply:
Do you want a unstable format to be part of the save format?
Should we discuss this in a separate issue? 🤔

Reply:
It's just as unstable inside the archive as outside of it, and we can always remove it from the archive using an mpq tool if we want to turn it in to a normal save, although it would work event if we didn't.

Reply:
We can do this (have a file that contains both the save game and the demo recording).
But I think this is outside of the scope of this pr, cause this pr only uses the currently demo format and don't introduce it.
Also the topic needs a little bit more thoughts, for example: If we have a `demo1.dmo` file that also includes the save game how do we know which version the save game belongs to (diablo, hellfire, spawn)?
I think this is all possible and a good idea but someone told me I should make small pr's. 😉

Reply:
![image](https://user-images.githubusercontent.com/204594/173235861-ec2f31c4-d60e-44e4-8dc6-37a8e8fad6be.png) ;)

We could check the header in `game` to see which game mode the save is for.
Or we could add it as part of the header line in the recording.

Reply:
With #4799 the most painful issue (large text files in the repro) is fixed.
I hope we can get the embedded demo file later.